packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -386,11 +399,11 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
     return _FlingGestureKind.reverse;
   }
 
-  void _handleDragEnd(DragEndDetails details) {
+  Future<void> _handleDragEnd(DragEndDetails details) async {
     if (!_isActive || _moveController.isAnimating)
       return;
     _dragUnderway = false;
-    if (_moveController.isCompleted) {
+    if (_moveController.isCompleted && await _confirmStartResizeAnimation() == true) {
```

Comment:
Why the explicit `== true` here and below?

Reply:
Both null and false mean false. Null is convenient because showDialog() returns null if the dialog is modal and the user taps outside of the dialog.